views/tenants/new.erb

```diff
@@ -1 +1,18 @@
-form for adding a new tenant
+<!-- form for adding a new tenant -->
+<h2>New Tenant</h2>
+
+  <form action="/tenant" method="post">
```

Comment:
You have forms inside of forms! You never want forms inside of forms. Just one form would work here:
```
<form action="/tenant" method="post">
  <input name="tenant[name]" />
...
  <input type="submit" value="Create" />
</form>
```